File: psup assignment/java/q7.java
import java.util.*;
class VoteCheck {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter age: ");
        int age = input.nextInt();
        if (age >= 18) {
            System.out.println("Eligible to vote.");
        } else {
            System.out.println("Not eligible to vote.");
        }
        input.close();
    }
}

File: psup assignment/cpp/q7.cpp
#include <iostream>
#include <string>
#include <cctype>
int main() {
    int age;
    std::cout << "\n--- Program 7: Voting Eligibility Check ---\n";
    std::cout << "Enter age: ";
    std::cin >> age;
    if (age >= 18) {
        std::cout << "Eligible to vote." << std::endl;
    } else {
        std::cout << "Not eligible to vote." << std::endl;
    }
    return 0;
}